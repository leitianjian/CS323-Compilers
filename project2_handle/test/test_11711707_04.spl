struct ST{
};

struct ST test(int n){
    struct ST s;
    struct ST s1;
    if(n < 0){
        return s;
    }else if ( n == 0){
        return s + s1;
    }else{
        return (s1);
    }
}