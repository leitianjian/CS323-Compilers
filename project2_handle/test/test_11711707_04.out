Error type 7 at Line 10: unsupported type of operands for arithmatic operator
Error type 8 at Line 10: function return type is not matched
